### DISCUSSION TITLE
Change /admin login view

### DISCUSSION THREAD

**Discussion by toto33331** [sulu/sulu#7075] [Q&A] [answered]
Hi everyone!

I'd like to change the login view but I have no idea how to do.
I've seen nowhere someone that changed the view.

**Do anyone know how I could do that ?**

**Comment by alexander-schranz** [comment, marked as answer]
There is no offical way to change the login view. You can just hook into the build process and overwrite the components: https://github.com/sulu/sulu-demo/pull/75

Keep in mind that on the Sulu updates you make sure to keep your changes uptodate with the changes of the Sulu Core Code.
